02-data.Rmd

```diff
@@ -1,18 +1,170 @@
+```{r}
+library(tidyverse)
+```
+
 # Data sources
 
-The data we are using for this project are listed as following:
+The data we use for this project consist of two datasets. NYC Parks Events data and NYC Parks Crimes data. NYC Parks Events dataset is acquired through the NYC OpenData  api. The data is collected by the Department of Parks and Recreation of the City of New York. The NYC Parks Crime dataset is acquired through the NYC.gov api. The data is provided by the New York Police Department. 
+
+You can use our prepared script at https://github.com/leoDYL/NYCEvents/blob/main/scripts/download_raw_data.R to download all the source raw data we used in this project. This script will create a `data/raw_data` directory under the project root (please run this scrip under the project root) and download all the source data files we used under this directory.
+
+## NYC Parks Events Data
+In the event listing datasets, which contains data from 2014 to 2018, we can take a look at the diversity of parks from the organizer, event category and time's perspectives. Based on the number of events hosted at different time and organized by different organizers, we can study on the liveliness of the parks. Other than that, since the datasets include detailed time information for each event from year to minute, we can study the time patterns that are presented in the data. Below, we introduce each data file for the park events. 
```

Comment:
start of 2013 to end of 2019 actually. But data for 2019 is certainly not complete, as discussed in [Timeseries Analysis on NYC Parks Events]

Reply:
Updated comment